src/mappings/staking.ts

```diff
@@ -162,6 +157,10 @@ export async function handleRemoveDelegation(
   );
 
   await delegation.save();
+
+  if (source === indexer) {
```

Comment:
why need to have this condition, if the source is delegator, we also need to update the indexer capacity